docs/integrations/mirroring.md

```diff
@@ -10,6 +10,9 @@ An example mirroring integration can be seen [here](https://github.com/demisto/c
 
 Mirroring integrations are developed the same as other integrations. They provide a few extra configuration parameters and APIs.
 
+:::note 
+Once an incident field is changed manually within XSOAR, it will be marked as "dirty" and will not be updated in XSOAR throughout the life of the incident. Furthermore, the field will still be mirrored out in case of any change of the incident in XSOAR.
```

Comment:
```suggestion
Once an incident field is changed manually within XSOAR, it will be marked as "dirty" and will not be updated in XSOAR throughout the life of the incident. It should be noted that in the case outbound mirroring is enabled, any changes to the incident in XSOAR will still be reflected in the external system.
```